packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -232,3 +235,10 @@ def log_context(logger: logging.Logger, level: int, msg: str, *args, **kwargs):
     logger.log(level, "Starting " + msg + " ...", *args, **kwargs)
     yield
     logger.log(level, "Finished " + msg, *args, **kwargs)
+
+
+def get_extra(input_dict: dict) -> dict | None:
```

Comment:
remaining like this should make the comment obsolete
```suggestion
def get_log_record_extra(input_dict: dict) -> dict | None:
```